Formula/siril.rb

```diff
@@ -18,6 +18,7 @@ class Siril < Formula
   depends_on "pkg-config" => :build
   depends_on "adwaita-icon-theme"
   depends_on "cfitsio"
+  depends_on "curl"
```

Comment:
macOS already includes curl. Doesn't it work with the system curl anymore?

Reply:
Indeed, you probably right.
I'm not a mac user so it is difficult for me to say that.
I remove this dep ?

Reply:
Please do

Reply:
@fxcoudert : done.